### DISCUSSION TITLE
Submission link

### DISCUSSION THREAD

**Discussion by maltanar** [Xilinx/brevitas-radioml-challenge-21#18] [Announcements]
As there are issues with the ITU submissions portal, please submit your solutions to the challenge through the following Google Form instead:

https://forms.gle/e5tu4UWHSh31SL5w7

Reminder: the submission deadline is **2021-10-09 AoE** (anywhere on Earth)

There is a 100 MB file size limit, if you need to submit something larger please get in touch with us directly.

**Comment by s59mz** [comment]
Are the winning models and solutions of this finished challenge publically available and where? I would like to test them if possible.
